Makefile

```diff
@@ -1,6 +1,6 @@
+VERSION := $(shell git describe --exact-match --tags 2> /dev/null || git rev-parse --short HEAD)
```

Comment:
This gives us tag only when `HEAD` is tagged.

```shell
> git describe --exact-match --tags
fatal: no tag exactly matches 'd12c2add1cf474f707e2edf7ce383ac7caf38e11'
> git tag @wemixkanvas/a@0.1.1
> git describe --exact-match --tags
@wemixkanvas/a@0.1.1
```

Please update this with below, this gives us the most recent tag that starts with `v`. Because we mix tagging with npm `@wemixkanvas/...`, and I don't want component to be affected by them

```diff
-VERSION := $(shell git describe --exact-match --tags 2> /dev/null || git rev-parse --short HEAD)
+VERSION := $(shell git describe --tags --abbrev=0 --match v* 2> /dev/null || git rev-parse --short HEAD)
```

Reply:
I also checked this working well inside the docker container, too.